codespell_lib/data/dictionary.txt

```diff
@@ -1341,6 +1344,9 @@ alllocation->allocation
 alllow->allow
 alllows->allows
 allmost->almost
+allcoate->allocate
+allcoated->allocated
+allcoating
```

Comment:
You missed out the corrected-version here??

Reply:
@lurch done @ 04eec2b4f5c5198bc97e08a230b096a934dd6dcb